Sources/StreamChat/Models/AttachmentTypes.swift

```diff
@@ -7,205 +7,14 @@ import Foundation
 /// A protocol for attachment model objects that are created from database `AttachmentDTO` objects and exposed.
 public protocol ChatMessageAttachment {
     var type: AttachmentType { get }
-    var id: AttachmentId? { get }
+    var id: AttachmentId? { get set }
```

Comment:
Why this change?

Reply:
Cause the object itself is decoded from the original payload and `id` is generated and set on the DB level.
Before introducing concrete types we were manipulating the `ChatMessageDefaultAttachment` type and the `id` property was `var` already.